src/luascript.cpp

```diff
@@ -12042,9 +12043,9 @@ int LuaScriptInterface::luaMonsterTypeGetCreatureEvents(lua_State* L)
 		return 1;
 	}
 
-	int index = 0;
-	lua_createtable(L, monsterType->scriptList.size(), 0);
-	for (const std::string& creatureEvent : monsterType->scriptList) {
+	int32_t index = 0;
```

Comment:
Revert this to int. It's only used to pass the index to lua_rawseti, which takes int as argument.